### ISSUE TITLE
A confusing warning about constructors

### ISSUE THREAD

**Issue by 4gn3s** [duaraghav8/Solium#9]
When a contract has a constructor, a warning informs the user, that it should follow the mixedCase convention:
```
contract Test {
    function Test() {
        /* my awesome constructor */
    }
}
```
and the warning:
```
[WARNING] At line: 2, column: 4 -> 'Test' doesn't follow the mixedCase notation
```

**Comment by duaraghav8** [comment]
The style guide says function names must follow mixedCase. But I forgot to add the exception for the Constructor.
This will be fixed soon:)

**Comment by duaraghav8** [comment]
solved in v0.1.9
